### DISCUSSION TITLE
Ability to Auto-Generate Schema Documentation

### DISCUSSION THREAD

**Discussion by bpetty-interlace** [sequelize/sequelize#15881] [Help & Questions]
We have some external and internal entities that would like to understand our database schema.
Externally, for different BI reporting purposes.  Internally, to get new developers up to speed on our design.
Are there any commands that could help facilitate this?  Or maybe a community project?
I would like to annotate the tables/models with a description as well that would be used in this process.

**Comment by ephys** [comment]
Some dialects let you write comments on tables & columns. You can do so using the `comment` option when initializing a model

```ts
Article.init({
  title: {
    type: DataTypes.STRING,
    comment: 'The Title',
  },
}, {
  sequelize,
  comment: 'The article table',
});
```
